client/src/components/History/HistoriesMenu.vue

```diff
@@ -1,37 +1,44 @@
 <template>
-    <PriorityMenu :starting-height="27">
-        <PriorityMenuItem
-            key="select-new-history"
-            title="Select History"
-            icon="fa fa-folder-open"
-            @click="$emit('selectHistoryModal')"
-        />
-        <PriorityMenuItem
-            key="create-new-history"
-            title="Create New History"
-            icon="fa fa-plus"
-            @click="$emit('createNewHistory')"
-        />
-        <PriorityMenuItem
-            key="view-all-histories"
-            title="View All Histories"
-            icon="fa fa-columns"
-            @click="redirect('/history/view_multiple')"
-        />
-        <PriorityMenuItem
-            key="saved-histories"
-            title="Saved Histories"
-            icon="fas fa-save"
-            @click="backboneRoute('/histories/list')"
-        />
-        <PriorityMenuItem key="clear-history-cache" title="Clear Cache" icon="fa fa-refresh" @click.stop="clearCache" />
-        <PriorityMenuItem
-            key="use-legacy-history"
-            title="Return to legacy history panel"
-            icon="fas fa-exchange-alt"
-            @click="switchToLegacyHistoryPanel"
-        />
-    </PriorityMenu>
+    <div id="current-history-panel">
```

Comment:
Is this div necessary?  It looks like this might collide with an existing `current-history-panel` identifier, as well.

Reply:
This was just for the selenium tests; to reuse some of the same methods that the legacy history panel uses.